requirements.txt

```diff
@@ -27,3 +27,4 @@ Unidecode==1.3.2
 urllib3==1.26.7
 visdom==0.1.8.9
 webrtcvad==2.0.10
+tkinter==8.6

```

Comment:
```suggestion

```

Reply:
I know I was the one that told you to add this, but it looks like tkinter is pre-installed on python, so it doesn't actually use pip to install.

Sorry about that!